docs/azure/sdk/resource-management.md

```diff
@@ -266,7 +265,7 @@ ArmClient armClient = new ArmClient(new DefaultAzureCredential());
 Subscription subscription = await armClient.GetDefaultSubscriptionAsync();
 string rgName = "myRgName";
 
-bool exists = await subscription.GetResourceGroups().CheckIfExistsAsync(rgName);
+bool exists = await subscription.GetResourceGroups().Exists(rgName).Value;
```

Comment:
`.Value` is not necessary as `Response<bool>` can be converted to `bool` implicitly here.

Reply:
@fengzhou-msft Other than that, you feel good about @xboxeer's changes? If so, I'll just make this one and merge.